src/main/java/frc/robot/subsystems/IntakeMechanisum.java

```diff
@@ -22,7 +22,7 @@ public IntakeMechanisum() {
     intake = new CANSparkMax(Constants.CANIDs.intake, MotorType.kBrushless);
     if (RobotBase.isSimulation()) {
 
-    REVPhysicsSim.getInstance().addSparkMax(intake, DCMotor.getNEO(1));
+  //  REVPhysicsSim.getInstance().addSparkMax(intake, DCMotor.getNEO(1));
```

Comment:
if we are commenting this out why not just delete it?

Reply:
Already done.